spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/channel/PostgresSubscribableChannel.java

```diff
@@ -39,19 +44,28 @@
  *
  * @author Rafael Winterhalter
  * @author Artem Bilan
+ * @author Igor Lovich
  *
  * @since 6.0
  */
 public class PostgresSubscribableChannel extends AbstractSubscribableChannel
 		implements PostgresChannelMessageTableSubscriber.Subscription {
 
+	private static final LogAccessor LOGGER = new LogAccessor(PostgresSubscribableChannel.class);
+
 	private final JdbcChannelMessageStore jdbcChannelMessageStore;
 
+	private TransactionTemplate transactionTemplate;
```

Comment:
Please, move this non-`final` property to respective section of props.

Reply:
done